pkg/context/transfer.go

```diff
@@ -17,6 +17,7 @@ import (
 // Transfer takes the logger, log.Sink, authentication, request and
 // error info from the given context and returns a complete
 // new context with all these objects.
+// Deprecated: Use context.WithoutCancel instead.
```

Comment:
I don't understand why this should be deprecated and instead we should user `context.WithoutCancel` 🤔

Reply:
I believe this function was introduced because one wanted to remove the cancellations and timeouts from context. That's at least my understanding by reading the comments above some of those calls. However, since Go 1.21 there is `context.WithoutCancel` which does exactly that.

Reply:
Hmm, ok makes.